src/go/rdctl/pkg/snapshot/manager_windows_test.go

```diff
@@ -81,7 +81,7 @@ func TestManagerWindows(t *testing.T) {
 				t.Fatalf("failed to modify %s: %s", testFileName, err)
 			}
 		}
-		if err := manager.Restore(snapshot.ID); err != nil {
+		if err := manager.Restore(snapshot.Name); err != nil {
```

Comment:
This seems unlikely to be in the correct PR…

Reply:
It's not, but unit tests won't pass on Windows without this change. But another piece of this fix is missing, so I'll pull  this out and create another PR. But unit tests won't pass on this PR.

Reply:
Pulled out, will create a separate PR